tests/test_operation.py

```diff
@@ -1197,3 +1197,107 @@ def test_qubit_state_vector_decomposition(self, monkeypatch):
         assert len(call_args) == 1
         assert np.array_equal(call_args[0][0], state)
         assert np.array_equal(call_args[0][1], wires)
+
+class TestChannel:
+    """Unit tests for the Channel class"""
+
+    def test_incorrect_num_wires(self):
+        """Test that an exception is raised if called with wrong number of wires"""
+
+        class DummyOp(qml.operation.Channel):
```

Comment:
It seems like these would address features of the `Operation` class (which are inherited by `Channel`). If no previous tests address them, it might be worth adding them simply for `Operation`. If they are tested for `Operation`, then these would be considered tested for `Channel` too (hence it's fine to not have explicit separate tests for `Channel`).

Reply:
oh okay. yes, many of these are already implemented for ``Operation``.